stories/ProductSearch.stories.js

```
@@ -23,50 +37,38 @@ export default {
 
 const Template = (args) => ({
     components: {
         VsProductSearch,
     },
     setup() {
+        // Mock the getData function
+        if (typeof window !== 'undefined') {
+            window.getData = mockGetData;
+        }
+        onUnmounted(() => {
+            delete window.getData;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)